pkgs/by-name/go/goshs/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "goshs";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "patrickhener";
+    repo = "goshs";
+    tag = "v${version}";
+    hash = "sha256-J74nUH3iBYDytXCswIexvShG5+zJEDX19saJ3ftLYhI=";
+  };
+
+  vendorHash = "sha256-6UaKh2UUyGqOriAaMhiEFr20R4W/ZWxQXtXnm/BVHOc=";
+
+  ldflags = [
```

Comment:
Why these ldflags?

Reply:
These two flags disable symbol generation and DWARF generation respectively and are AFAICT fairly standard for release-build Go packages. No need to strip them out later if we never generate them to begin with